### PR TITLE
Improve README to note that submodule dependencies must be satisfied …

### PR DESCRIPTION
**Pull request by thepurpleblob**
…if deploying direct from Github repo. Issue 132

### CODE REVIEW

**Comment by garemoko** [comment]
Huh. I didn't realize we had two readme files. Please can you make all the changes in readme.md and either delete readme.txt in this PR or I can do it after.

File: README.txt [hunk 1]
```diff
@@ -48,6 +50,17 @@ from a previous version of this plugin, and you don't know how to fix the upgrad
...
 Note that this workaround will delete any instances of the plugin that you have set up in your courses.
 
+##Installation from Github
+
+This is recommended only for developers or if you need the very latest versions. 
+
+The plugin has two submodule dependencies and will not work directly from a clone. Go into the plugin folder
+and type the command...
+
+git submodule update --init --recursive
```
Comment (garemoko) [inline README.txt:60]: Please put this in a markdown code block. Thanks!

**Review by garemoko — CHANGES_REQUESTED** [review]

File: README.txt (hunk 1 above)
Comment (thepurpleblob) [reply README.txt:60]: What about all the rest of the stuff in README.txt... do you still want that?

Comment (garemoko) [reply README.txt:60]: Sorry, just saw this. No, that other stuff was deliberately deleted because its out of date.

**Pull request merged**